documentation/modules/advanced/pages/kiali.adoc

```diff
@@ -32,21 +36,29 @@ Install Kiali with the following commands:
 
 [source, bash]
 ----
-# This command installs Kiali's configmaps
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml | \
-   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
-
-# This command installs Kiali's secrets
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-secrets.yaml | \
-   VERSION_LABEL=master envsubst | oc create -n istio-system -f -
-
-# And this command installs the Kiali service
-curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
-   IMAGE_NAME=kiali/kiali \
-   IMAGE_VERSION=latest \
-   NAMESPACE=istio-system \
-   VERSION_LABEL=master \
-   VERBOSE_MODE=4 envsubst | oc create -n istio-system -f -
+# URLS for Jaeger and Grafana
+JAEGER_URL="https://jaeger-query-istio-system.$(minishift ip).nip.io"
+GRAFANA_URL="https://grafana-istio-system.$(minishift ip).nip.io"
+VERSION_LABEL="v0.7"
+
+# Installs Kiali's configmap
+curl https://raw.githubusercontent.com/kiali/kiali/${VERSION_LABEL}/deploy/openshift/kiali-configmap.yaml | \
+  VERSION_LABEL=${VERSION_LABEL} \
+  JAEGER_URL=${JAEGER_URL}  \
+  GRAFANA_URL=${GRAFANA_URL} envsubst | oc create -n istio-system -f -
```

Comment:
Can you add at requirements page that `envsubst` is required and how to install it?